android-components/components/browser/menu/src/main/res/layout/mozac_browser_menu_item_image_text_checkbox_button.xml

```diff
@@ -10,7 +10,8 @@
     android:focusable="true"
     android:gravity="center_vertical"
     android:orientation="horizontal"
-    tools:ignore="UseCompoundDrawables">
+    android:paddingEnd="0dp"
```

Comment:
Could you explain what this change is doing? Why not paddingHorizontal?

Reply:
Now, reading your other comment, I realize that probably I should be doing that by overriding the dimens as well. 

But anyway, intent is to remove end container spacing, and delegate the end spacing for the checkbox button.
That's the structure of spacing that we have now: 
`> >text_view< >checkbox< <`. 

And that's what I am going for:
`> >text_view< >checkbox<`.